city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,178 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+    agenda_map = defaultdict(list)
```

Comment:
Could you explain this a bit more? We've usually been initializing `defaultdict` objects in the `__init__` method to avoid any complications of when specifically they spider object is created. It also looks like there's a getter method in `_get_agenda` that could be avoided unless I'm missing something in the implementation